app/dashboard/static/js/app/view-test-plan-all.2020.1.20.js

```diff
@@ -0,0 +1,532 @@
+        }
+
+        if (results[0].git_commit) {
+            html.replaceContent(
+                document.getElementById('git-commit'),
+                document.createTextNode(results[0].git_commit));
+        } else {
+            html.replaceContent(
+                document.getElementById('git-commit'), html.nonavail());
+        }
+
+        if (results[0].status) {
+            html.replaceContent(
+                document.getElementById('git-status'),
+                document.createTextNode(results[0].git_status));
+        } else {
+            html.replaceContent(
+                document.getElementById('git-status'), html.nonavail());
+        }
+
+        if (results[0].cross_compile) {
+            html.replaceContent(
+                document.getElementById('cross-compile'),
+                document.createTextNode(results[0].cross_compile));
+        } else {
+            html.replaceContent(
+                document.getElementById('cross-compile'), html.nonavail());
+        }
+
+        if (results[0].defconfig) {
+            html.replaceContent(
+                document.getElementById('build-defconfig'),
+                document.createTextNode(results[0].defconfig));
+        } else {
+            html.replaceContent(
+                document.getElementById('build-defconfig'), html.nonavail());
+        }
+
+        if (results[0].compiler_version) {
+            html.replaceContent(
+                document.getElementById('compiler-version'),
+                document.createTextNode(results[0].compiler_version));
+        } else {
+            html.replaceContent(
+                document.getElementById('compiler-version'),
+                html.nonavail());
+        }
+
+        if (results[0].compiler_version_full) {
+            html.replaceContent(
+                document.getElementById('compiler-version-full'),
+                document.createTextNode(results[0].compiler_version_full));
+        } else {
+            html.replaceContent(
+                document.getElementById('compiler-version-full'),
+                html.nonavail());
+        }
+
+        if (results[0].arch) {
+            html.replaceContent(
+                document.getElementById('build-arch'),
+                document.createTextNode(results[0].arch));
+        } else {
+            html.replaceContent(
+                document.getElementById('build-arch'), html.nonavail());
+        }
+
+        html.replaceContent(
+            document.getElementById('build-errors'),
+            document.createTextNode(results[0].errors));
+
+        html.replaceContent(
+            document.getElementById('build-warnings'),
+            document.createTextNode(results[0].warnings));
+    }
+
+    function getTestPlans() {
+        var data;
+        var deferred;
+        data = {
+            tree: gTree,
+            git_branch: gBranch,
+            //distinct: 'name',
```

Comment:
@gctucker seems that this query filtering by `kernel`, `branch` and `tree` don't match anything. Is distinct available for `test_group`? It is important for test plan list..because we need to generate unique ids to update the table in order to put the numbers.

Reply:
See how I did this in the view I made:
https://github.com/gctucker/kernelci-frontend/blob/T18074-test-plans-view/app/dashboard/static/js/app/view-tests-job-branch-kernel.2020.1.js#L279

Reply:
sure, you aggregated the data :)